### PR TITLE
Update CDC Vaccine Exclusions

### PR DESCRIPTION
**Pull request by smcclure17**
Updating CDC Vaccine blocks as the result of the review here: https://www.dropbox.com/scl/fi/rtbyfq9pfvnj2w1fki8fg/02_02_2022_-CDC-Vaccine-data-review.paper?dl=0&rlkey=pq42hko1hxcx6l9ga86bb6f2t

### CODE REVIEW

File: libs/datasets/combined_datasets.py [hunk 1]
```diff
@@ -196,29 +196,24 @@ def datasource_regions(
...
         "PA",
-        # SD - Missing a lot of counties and 1st dose data.
-        "SD",
         # VA - Very low coverage.
         "VA",
         "VT",
-        "WV",
         # CDC reporting 0.1% vaccinated
         "HI",
-        # TODO(sean) 10/23/21: Block TX CDC data until we have a chance to properly QA it
```
Comment (smcclure17) [inline libs/datasets/combined_datasets.py:220]: a couple of months late here sorry :/

File: libs/datasets/combined_datasets.py [hunk 2]
```diff
@@ -196,29 +196,24 @@ def datasource_regions(
     Region.from_fips("02060"),
     # Culebra Municipio, PR - reports 99.9% of 12+ population vaccinated [7/2021]
     Region.from_fips("72049"),
+    # Chattahoochee County, GA - reports well over 100% 1+ dose vaccinated [2/2022]
+    Region.from_fips("13053"),
```
Comment (smcclure17) [inline libs/datasets/combined_datasets.py:200]: If we'd prefer to not mix sources within a state/region we can remove these county exclusions and potentially look to block these locations instead

Comment (mikelehen) [reply libs/datasets/combined_datasets.py:200]: I'm a bit on the fence.  Let's go ahead with keeping them for now, I guess.  I get more nervous when there are a lot of regions in a state or they're the biggest counties.  But small one-off counties 🤷

File: libs/datasets/combined_datasets.py (hunk 1 above)
Comment (mikelehen) [reply libs/datasets/combined_datasets.py:220]: 🤣

**Review by mikelehen — APPROVED** [review]
Sorry for the delay!

**Pull request merged**